src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -464,13 +465,20 @@ struct GhInterfaceManager : db::SimpleTag {
 /// provide data for CCE.
 struct InterfaceManagerInterpolationStrategy : db::SimpleTag {
   using type = InterfaceManagers::InterpolationStrategy;
-  using option_tags = tmpl::list<OptionTags::GhInterfaceManager>;
+  using option_tags = tmpl::list<OptionTags::GhInterfaceManager,
+                                 ::OptionTags::TimeStepper<TimeStepper>>;
 
   static constexpr bool pass_metavariables = false;
-  static InterfaceManagers::InterpolationStrategy
-  create_from_options(
+  static InterfaceManagers::InterpolationStrategy create_from_options(
       const std::unique_ptr<InterfaceManagers::GhInterfaceManager>&
-          interface_manager) noexcept {
+          interface_manager,
+      const std::unique_ptr<TimeStepper>& time_stepper) noexcept {
+    ASSERT(interface_manager->get_interpolation_strategy() !=
+                   InterfaceManagers::InterpolationStrategy::EveryStep or
+               dynamic_cast<LtsTimeStepper*>(time_stepper.get()) != nullptr,
+           "The use of full-step interface managers (which perform "
+           "interpolation and extrapolation between the CCE and GH systems) "
+           "require an LTS-compatible time-stepper.");
```

Comment:
Do you need LTS, or specifically AB?

Also, this looks like a user error, so ASSERT -> ERROR.

Reply:
I think the constraint I require is that the times of the steps are strictly increasing, and I had the impression that LTS steppers required the same thing -- if those requirements are different, though, I should just require AB specifically.

Agreed, I'll make it `ERROR` instead.

Reply:
(This is all somewhat academic, since AB is the only implementation of LtsTimeStepper and there are no plans to add any more in the near future.)  Interesting question.  If you don't care about conservation, you can certainly use non-monotonic steps, but I doubt we will ever add a non-conservative stepper.  The only other LTS ideas I have considered are AB-like, and if they are added they should still work for you, so LTS is probably better here.

But maybe what you really want is `number_of_substeps() == 1`.